.github/workflows/drivers_ci.yml

```diff
@@ -0,0 +1,188 @@
+name: Drivers CI Build
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release/**'
+      - 'maintainers/**'
+  workflow_dispatch:
+  # we cannot use paths key here since otherwise required_status_checks jobs won't run.
+  # See https://github.com/orgs/community/discussions/26251.
+  # We need to use the paths-filter job.
+
+# Checks if any concurrent jobs under the same pull request or branch are being executed
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  paths-filter:
+    runs-on: ubuntu-latest
+    outputs:
+      driver_changed: ${{ steps.filter.outputs.driver }}
+      libscap_changed: ${{ steps.filter.outputs.libscap }}
+      libsinsp_changed: ${{ steps.filter.outputs.libsinsp }}
+    steps:
+    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+    - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # v2.11.1
+      id: filter
+      with:
+        filters: |
+          driver:
+            - 'driver/**'
+          libscap:
+            - 'userspace/libscap/**'
+          libsinsp:
+            - 'userspace/libsinsp/**'
+
+  # This job run all engine tests and scap-open
+  test-scap:
+    name: test-scap-${{ matrix.arch }} 😆 (bundled_deps)
+    runs-on: ${{ (matrix.arch == 'arm64' && 'actuated-arm64-8cpu-16gb') || 'ubuntu-22.04' }}
+    needs: paths-filter
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    steps:
+      - name: Checkout Libs ⤵️
+        if: needs.paths-filter.outputs.driver_changed == 'true' || needs.paths-filter.outputs.libscap_changed == 'true'
```

Comment:
We need to skip each step because of https://github.com/orgs/community/discussions/9141.
This avoids having a skipped job whose name is not the resolved matrix name but instead:
> CI Build / build-drivers-${{ matrix.arch }} :smile: : (system_deps) (pull_request) Skipped

thus breaking the required  status checks once again.